dotcom-rendering/src/web/components/Card/components/TrailTextWrapper.tsx

```diff
@@ -7,32 +7,59 @@ type Props = {
 	children: string | React.ReactNode;
 	format: ArticleFormat;
 	containerPalette?: DCRContainerPalette;
+	imagePosition?: ImagePositionType;
+	imageSize?: ImageSizeType;
+	imageType?: CardImageType | undefined;
+};
+
+const showTrailText = (
+	imagePosition?: ImagePositionType,
+	imageSize?: ImageSizeType,
+	imageType?: CardImageType | undefined,
+) => {
+	if (
+		imageSize === 'large' &&
+		imagePosition === 'right' &&
+		imageType !== 'avatar'
+	)
+		return css`
+			${until.desktop} {
+				display: none;
+			}
+		`;
+	return css`
+		${until.tablet} {
+			display: none;
+		}
+	`;
 };
 
 export const TrailTextWrapper = ({
 	children,
 	format,
 	containerPalette,
+	imagePosition,
+	imageSize,
+	imageType,
 }: Props) => {
 	const palette = decidePalette(format, containerPalette);
 	return (
 		<div
-			css={css`
-				display: flex;
-				flex-direction: column;
-				color: ${palette.text.cardStandfirst};
-
-				${body.small({ lineHeight: 'regular' })};
-				font-size: 14px;
+			css={[
+				css`
+					display: flex;
+					flex-direction: column;
+					color: ${palette.text.cardStandfirst};
 
-				padding-left: 5px;
-				padding-right: 5px;
-				padding-bottom: 8px;
+					${body.small({ lineHeight: 'regular' })};
+					font-size: 14px;
```

Comment:
These declaration seem at odd. Either we want `body.small` or we want a different size. Would `xsmall` be more adequate?